Homogeneous_kernel/include/CGAL/Homogeneous/Aff_transformationH3.h

```diff
@@ -82,11 +82,19 @@ class Aff_transformation_rep_baseH3 : public Ref_counted_virtual
   virtual  bool
            is_even() const = 0;
 
+  virtual  bool
+           is_translation() const { return false; }
+
+  virtual  bool
+           is_scaling() const { return false; }
+
   virtual  RT
            homogeneous(int i, int j) const = 0;
 
   virtual  FT
            cartesian(int i, int j) const = 0;
+
+  virtual  Aff_transformation_3 compose(const Aff_transformation_rep_baseH3* aff) const { return Aff_transformation_3(); }
```

Comment:
warning: `aff` is not used here

Reply:
if not used, shouldn't it be pure virtual?

Reply:
I think it intentionally still has an implementation @sloriot